projects/aca-content/folder-rules/src/folder-rules.rules.ts

```diff
@@ -22,10 +22,10 @@
  * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { AcaRuleContext, hasFolderSelected, canEditFolder, isNotFavorites } from '@alfresco/aca-shared/rules';
+import { AcaRuleContext, canEditFolder, hasFolderSelected, isNotFavorites, isSmartFolder } from '@alfresco/aca-shared/rules';
 
 export const isFolderRulesEnabled = (context: AcaRuleContext) => context.appConfig.get<boolean>('plugins.folderRules', false);
 export const isFolderRulesAllowed = (context: AcaRuleContext) =>
-  isFolderRulesEnabled(context) && canEditFolder(context) && hasFolderSelected(context) && isNotFavorites(context);
+  isFolderRulesEnabled(context) && canEditFolder(context) && hasFolderSelected(context) && isNotFavorites(context) && !isSmartFolder(context);
```

Comment:
could you add unit tests to projects/aca-content/folder-rules/src/folder-rules.rules.spec.ts file?

Reply:
@AleksanderSklorz I have already, there is this case with smart folder added by me

Reply:
Ah, I see, thanks :)